lib/configs/routes.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:flutter/widgets.dart';
+import 'package:survey/screens/landing/landing_screen.dart';
+
+final Map<String, WidgetBuilder> routes = {
+  "/": (_) => LandingScreen()
+};
```

Comment:
curious if we need to add the end of file line or not, maybe you want to run `flutter fmt` to check